mlir-tensorrt/python/bindings/Runtime/RuntimePyBind.cpp

```diff
@@ -119,6 +120,16 @@ struct type_caster<MTRT_MemRefValue> {
 
 namespace {
 
+bool isGPUPointer(void *ptr) {
```

Comment:
According to LLVM style guide, functions that are translation-unit local should be declared 'static' as opposed to being located in anonymous namespaces. class/structs that are local to the TU are wrapped in anonymous namespaces 

https://llvm.org/docs/CodingStandards.html#anonymous-namespaces